src/i18n/de-de/index.js

```diff
@@ -0,0 +1,358 @@
+            invalid_signed_integer: 'Ungültiger signed integer',
+            invalid_unsigint: 'Der Eintrag muss einen gültigen unsigned integer enthalten',
+            str_input_placeholder: '["some value", ... , "final value"]',
+            str_input_hint: 'Doppelte Anführungszeichen in Strings müssen escaped werden (\\")',
+            address_placeholder: 'Adresse beginnt mit 0x',
+            address_label: '{ label } (Adresse/Konto)',
+            boolean_array_label: '{ label } (bool[{ size }])',
+        },
+        health: {
+            status: 'Status',
+            checked_at: 'Geprüft am',
+            task: 'Aufgabe',
+            message: 'Nachricht',
+            block_height: 'Block Nummer',
+            latency: 'Latenz',
+            click_to_change_format: 'Zum Ändern des Formats anklicken',
+        },
+        contract_tab: {
+            copy_abi_to_clipboard: 'ABI JSON des Smart Contracts in die Zwischenablage kopieren',
+            enter_amount: 'Wählen Sie Anzahl der Dezimalstellen und geben Sie einen Betrag ein, dieser wird als Funktionsparameter als uint256 erfasst',
+            result: 'Ergebnis',
+            view_transaction: 'Transaktion ansehen',
+            code: 'Code',
+            read: 'Lesen',
+            write: 'Schreiben',
+            amount: 'Anzahl',
+            value: 'Wert',
+            custom_decimals: 'Benutzerdefinierte Dezimalen',
+            custom: 'Benutzerdefiniert',
+            unverified_contract_source: 'Dieser Smart Contract wurde nicht überprüft..',
+            click_here: 'Klicken Sie hier, ',
+            upload_source_files: 'um Quelldateien hochzuladen und diesen Smart Contract zu überprüfen. ' +
+                'Alternativ können Sie mit dem Smart Contract interagieren, indem Sie ein beliebiges ABI verwenden:',
+            use_erc20_abi: 'ERC20 ABI nutzen',
+            use_erc721_abi: 'ERC721 ABI nutzen',
+            abi_from_json: 'ABI aus JSON-Datei',
+            paste_abi_json_here: 'ABI JSON hier einfügen',
+            abi_json_preview: 'ABI JSON Vorschau',
+            provided_abi_invalid: 'Bereitgestellte ABI ist ungültig',
+            provided_json_invalid: 'Bereitgestellte JSON ist ungültig',
+            read_functions: 'Lese-Funktionen',
+            write_functions: 'Schreib-Funktionen',
+            unverified_contract: 'Ungeprüfter Smart Contract',
+        },
+        header: {
+            sign_in: 'Anmelden',
+            sign_out: 'Abmelden',
+            liq_staking: 'Liquid Staking / flexible Anlage',
+            goto_staking: 'Staking/Anlage öffnen',
+            goto_health_monitor: 'Heath Monitor öffnen',
+            goto_mainnet: 'Mainnet auswählen',
+            goto_testnet: 'Testnet auswählen',
+            health_monitor: 'Health Monitor',
+            health_status: 'Health Status',
+            advanced: 'Erweitert',
+            light_mode: 'Light Mode',
+            dark_mode: 'Dark Mode',
```

Comment:
should these be kept in English?

Reply:
yes, same as some other words the translation would be pretty clunky.